ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -574,6 +574,15 @@ private function checkLinkCondition(array $item, array $data): bool {
       }
       return TRUE;
     }
+    // Convert the conditional value of 'current_domain' into an actual value that filterCompare can work with
+    if (substr($item['condition'][0], 0, 9) === 'domain_id' && $item['condition'][2] === 'current_domain') {
+      if (strpos($item['condition'][0], 'label') !== FALSE) {
```

Comment:
Shouldn't this be
```suggestion
      if (str_ends_with($item['condition'][0], ':label')) {
```

Reply:
Updated to that, I didn't know about the str_ends_with

Reply:
It's new in PHP8, but Civi includes shims for older php versions so we're allowed to use it :)